### ISSUE TITLE
give users a way to squelch individual warnings

### ISSUE THREAD

**Issue by searls** [testdouble/testdouble.js#90]
Give users a local way to squelch warnings generated by the library. Ideally this should be local to either each rule or to each violation, to discourage users from globally disabling all warnings.

Hey @jasonkarns &  @davemo any opinions on this?

One idea:

```js
td.ignoreWarnings('optional excuse here', function(){
  td.when(somethingThatWouldTriggerAWarning()).thenReturn()
  //…
})
```

**Comment by davemo** [comment]
What about just an internal configuration flag that's less granular than scoping to a particular block?

Maybe expose it through something like `td.config({ignoreWarnings: false})` where `options` contains `{ignoreWarnings: false}` as a default?

**Comment by davemo** [comment]
Then you could just put the `td.config` in your `test_helper` file once and be done with it.

**Comment by searls** [comment]
I'd prefer not to give people only a global option for disabling all warnings when what's most likely is they have one or two they have good reason for generating

**Comment by searls** [comment]
@davemo wins this one:

```js
td.config({
  ignoreWarnings: true
})
```

Just landed in 1.3.0

**Comment by davemo** [comment]
:metal:
